kolibri/core/assets/src/vue/nav-bar/index.vue

```diff
@@ -296,6 +298,10 @@
     fill: $core-bg-light
 
   .header
+    position: absolute
+    z-index: 1003
```

Comment:
whoa

we should really try and standardize this whole z index thing... and document it somewhere